HoloLensCommander/HoloLensCommander/Utilities/WebRequest.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Threading.Tasks;
+using Windows.Web.Http;
+using Windows.Web.Http.Headers;
+
+namespace HoloLensCommander
+{
+    public class AppCenterWebRequest
+    {
+        public async Task<string> HttpWebRequest()
+        {
+            HttpClient httpClient = new HttpClient();
+
+            var request = new HttpRequestMessage();
+
+            request.Headers.Accept.Add(new HttpMediaTypeWithQualityHeaderValue("application/json"));
+            request.Headers.Append("X-API-Token", "9b8e356d29f80de93d88fac0252e48f8fbe22e96");
```

Comment:
We may want to read this from settings (with a const variable that defines the currently known value) just in case it changes when the API / endpoint is revised,

Reply:
@davidkline-ms I intend to save/read the token from localcache. What are best practices for defining API endpoints in the app and reading them from settings?

Reply:
see below